src/UrlChecker/UrlCheckerTrait.php

```diff
@@ -54,7 +55,8 @@ protected function _getLoginUrlChecker()
             $options['className'] = DefaultUrlChecker::class;
         }
 
-        $checker = new $options['className']();
+        $className = App::className($options['className'], 'UrlChecker', 'UrlChecker');
```

Comment:
Since we've built the plugin with being useable in other apps / frameworks, will this line work OK outside the context of a CakePHP application? Is it even then aware of the correct paths? I'm not sure, never done it, that's why I'm asking.

Reply:
The `cakephp/core` package is required so this will work outside the framework. The support for `Plugin.Class` notation is something we use throughout this plugin. Note that authenticators and identifiers are loaded this way: `$service->loadAuthenticator('Authentication.Form');` etc